static/js/components/dashboard/FinalExamCard.js

```diff
@@ -267,28 +267,55 @@ export default class FinalExamCard extends React.Component<void, Props, void> {
       },
       showPearsonTOSDialog
     } = this.props
-
-    switch (program.pearson_exam_status) {
-    case PEARSON_PROFILE_ABSENT:
-      return absentCard()
-    case PEARSON_PROFILE_SUCCESS:
-      return successCard(profile, navigateToProfile)
-    case PEARSON_PROFILE_IN_PROGRESS:
-      return pendingCard()
-    case PEARSON_PROFILE_INVALID:
-      return invalidCard(navigateToProfile)
-    case PEARSON_PROFILE_SCHEDULABLE:
-      return schedulableCard(
-        profile,
-        program,
-        navigateToProfile,
-        pearson,
-        showPearsonTOSDialog,
-        pearsonTOSDialogVisible,
-        submitPearsonSSO
+    if (!SETTINGS.FEATURES.ENABLE_EDX_EXAMS) {
+      switch (program.pearson_exam_status) {
+      case PEARSON_PROFILE_ABSENT:
+        return absentCard()
+      case PEARSON_PROFILE_SUCCESS:
+        return successCard(profile, navigateToProfile)
+      case PEARSON_PROFILE_IN_PROGRESS:
+        return pendingCard()
+      case PEARSON_PROFILE_INVALID:
+        return invalidCard(navigateToProfile)
+      case PEARSON_PROFILE_SCHEDULABLE:
+        return schedulableCard(
+          profile,
+          program,
+          navigateToProfile,
+          pearson,
+          showPearsonTOSDialog,
+          pearsonTOSDialogVisible,
+          submitPearsonSSO
+        )
+      default:
+        return null
+      }
+    } else {
+      return (
+        <Card className="card final-exam-card">
+          <CardContent>
+            <div className="card-header">
+              <div>
+                <img className="exam-icon" src="/static/images/exam_icon.png" />
+              </div>
+              <div className="exam-text">
+                <h2>Final Proctored Exam</h2>
+                <p>
+                  To earn a certificate, you must take an online proctored exam
+                  for each course. Before you can take a proctored exam, you
+                  have to pay for the course and pass the online work.
+                </p>
+                <p>
+                  Exams will be available online on edX.org. You may take the
```

Comment:
Should `edX.org` be a link?

Reply:
no the link will come from a a csv file. This work hasn't been done yet.